src/compton_tools/Sparse_Compton_Matrix.hh

```diff
@@ -0,0 +1,101 @@
+//--------------------------------------------*-C++-*---------------------------------------------//
+/*!
+ * \file   compton_tools/Sparse_Compton_Matrix.hh
+ * \author Andrew Till
+ * \date   22 May 2020
+ * \brief  Header file for compton interface
+ * \note   Copyright (C) 2017-2020 Triad National Security, LLC. All rights reserved.
+ */
+//------------------------------------------------------------------------------------------------//
+
+#ifndef rtt_compton_tools_Sparse_Compton_Matrix_hh
+#define rtt_compton_tools_Sparse_Compton_Matrix_hh
+
+#include <vector>
+
+namespace rtt_compton_tools {
+//------------------------------------------------------------------------------------------------//
+/*!
+ * \class Sparse_Compton_Matrix
+ *
+ * \brief Stores a Compton scattering matrix sparsely and allows
+ *        matrix multiplication; can store an arbitrary number of points,
+ *        where each point represents an  M x G x G matrix (moments, groups)
+ *
+ *        THIS CLASS IS A STUB (not yet fully implemented or used)
```

Comment:
If this isn't needed, then maybe is shouldn't be in the PR?  Or maybe you can leave it in a `#ifdef 0` block for now.

Reply:
Yeah, I'll add `#ifdef 0`